testapi.pm

```diff
@@ -455,7 +455,9 @@ needle area. If C<$dclick> is set, do double click instead.  C<$mustmatch> can
 be string or C<ARRAYREF> of strings (C<['tag1', 'tag2']>).  C<$button> is by
 default C<'left'>. C<'left'> and C<'right'> is supported. If C<$mousehide> is
 true then always move mouse to the 'hidden' position after clicking to prevent
-to hide the area where user wants to assert/click in second step.
+to hide the area where user wants to assert/click in second step. If
```

Comment:
```suggestion
to hide the area where the user wants to assert/click in the second step. If
```

Reply:
This and the above suggestion both came from copy/pasting existing things within the files, so I'll adjust the places I took them from as well.